process/best_practices_badge.md

```diff
@@ -34,29 +121,143 @@ and in OSL:
+
+### The project MUST document what the user can and cannot expect in terms of security from the software produced by the project (its security requirements).
+
+This likely will align with the security assurance case, but the doc should cover what the project does for security ( for example, input sanitization ) and areas that the project doesn't directly address but the user should ( for example, don't run as `root` ).
+
+### The project MUST provide a quick start guide for new users to help them quickly do something with the software.
+
+This is a cornerstone of good documentation. Part of this is the installation instructions, but having an example to show how the project works or something the user could do to understand how it works is of huge value to prospective users. OpenEXR has a [great example](https://github.com/AcademySoftwareFoundation/openexr#quick-start).
+
+### The project MUST make an effort to keep the documentation consistent with the current version of the project results (including software produced by the project). Any known documentation defects making it inconsistent MUST be fixed. If the documentation is generally current, but erroneously includes some older information that is no longer true, just treat that as a defect, then track and fix as usual.
+
+Good practice for any project. Many projects keep the documentation alongside the code so when changes are made it's reflected in the same branch. Just The Docs has support for multiple versions of documentation.
+
+### The project repository front page and/or website MUST identify and hyperlink to any achievements, including this best practices badge, within 48 hours of public recognition that the achievement has been attained.
+
+Put it in the README.md, either as a badge or textual notice.
+
+### The project (both project sites and project results) SHOULD follow accessibility best practices so that persons with disabilities can still participate in the project and use the project results where it is reasonable to do so.
+
+If a project is just working out of GitHub/Read The Docs/Confluence and not building a formal website, then this requirement is fulfilled. If you have a separate website, it's a good idea to review and assess.
+
+### The software produced by the project SHOULD be internationalized to enable easy localization for the target audience's culture, region, or language. If internationalization (i18n) does not apply (e.g., the software doesn't generate text intended for end-users and doesn't sort human-readable text), select not applicable (N/A).
+
+This only applies if there is text generated by the project for the user; if not n/a. Note you don't need multiple languages supported, just the mechanism such that you could ( i.e., split text strings out to a separate document ).
```

Comment:
The first sentence here seems to just restate the last sentence. Do people not understand the criterion last sentence?

You're absolutely right, you don't need to localize for multiple language, just create the i18n mechanism.